packages/flutter/lib/src/widgets/app.dart

```diff
@@ -23,15 +25,51 @@ import 'widget_inspector.dart';
 
 export 'dart:ui' show Locale;
 
+/// The signature of [WidgetsApp.localeListResolutionCallback].
+///
+/// A `LocaleListResolutionCallback` is responsible for computing the locale of the app's
```

Comment:
use square brackets, not backticks, to refer to things that can be cross-referenced (even if it's a self-cross-reference, that way copy-and-paste later works better).